apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -48,7 +61,17 @@ export default function ListBoxSearch({ selections, model, keyboard, dense = fal
     };
   }, []);
 
+  useEffect(() => {
+    if (wildcardOn) {
+      const cursorPos = value.length - 1;
+      input.setSelectionRange(cursorPos, cursorPos); // place the cursor in the wildcard
+    }
```

Comment:
Move the setWildcardOn -> false to this place and should only run at focus and never again.
```suggestion
      setWildcardOn(false);
    }
```